recipes/bcbio-nextgen/meta.yaml

```diff
@@ -3,7 +3,7 @@ package:
   version: '1.0.8a'
 
 build:
-  number: 3
+  number: 4
```

Comment:
pinning on run-only dep can be removed (needs https://github.com/bioconda/bioconda-utils/pull/264 to make linter happy)